.circleci/config.yml

```diff
@@ -85,4 +99,13 @@ workflows:
             tags:
               only: /.*/ # allow anything because tag syntax is validated as part of validate-release.sh
             branches:
-              ignore: /.*/
\ No newline at end of file
+              ignore: /.*/
+      - containers_publish:
+          requires:
+            - test_examples
```

Comment:
I think this should also require `- npm_publish` considering that the new images will need to have the new packages published to the registry before we can install them as part of the container build. We may need to test that one out because sometimes it takes some time before published packages become available on the registry.

Reply:
done